packages/image_picker/example/lib/main.dart

```diff
@@ -249,8 +265,67 @@ class _MyHomePageState extends State<MyHomePage> {
     }
     return null;
   }
+
+  Future<void> _displayPickImageDialog(
+      BuildContext context, OnPickImageCallback onPick) async {
+    return showDialog(
+        context: context,
+        builder: (context) {
+          return AlertDialog(
+            title: Text('TextField in Dialog'),
```

Comment:
nit: This title isn't really descriptive. "Set image size" maybe?

Reply:
good find! Thanks. I updated it `Add optional parameters`